### PR TITLE
Adds the BSG to RND designs

### PR DESCRIPTION
**Pull request by Qwertytoforty**
## What Does This PR Do

Adds a design for the BSG to rnd, to actually make it producible, like intended. Has heavy RND costs.

## Why It's Good For The Game

Actually being able to craft the gun in RND, like intended, is a good thing. Fixes me being dumb and not giving it a RND design.

## Changelog
:cl:
fix: Fixes the BSG not having a design in the protolathe.
/:cl:

### CODE REVIEW

File: code/modules/research/designs/weapon_designs.dm [hunk 1]
```diff
@@ -269,3 +269,14 @@
 	locked = TRUE
 	access_requirement = list(ACCESS_RD)
 	category = list("Weapons")
+
+/datum/design/bsg
+	name = "Blue Space Gun"
+	desc = "A heavy hitting energy cannon, that fires destructive bluespace blasts with a decient area of effect."
```
Comment (SteelSlayer) [inline code/modules/research/designs/weapon_designs.dm:275]:
```suggestion
	desc = "A heavy hitting energy cannon, that fires destructive bluespace blasts with a decent area of effect."
```

**Review by SteelSlayer — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]
Code approved, not sure about design though.

Yes its expensive, but if theres competent miners, there could be 3 or 4 of these.

**Comment by hal9000PR** [comment]
You still need the flux core to make it no?

**Comment by AffectedArc07** [comment]
EDIT: Given the core thing, this is probably fine

**Comment by LightFire53** [comment]
It's called Blue Space Gun, even though Bluespace is all a single word usually?

**Comment by hal9000PR** [comment]
> It's called Blue Space Gun, even though Bluespace is all a single word usually?

This is because it's a reference to the BFG.

**Review by Kyep — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]
R&D Division? Nah, https://www.youtube.com/watch?v=W9GaIbECisQ

**Pull request merged**